foreman_kernel_care.gemspec

```diff
@@ -15,10 +15,9 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.5.0'
 
-  s.add_dependency 'foreman_remote_execution', '>= 1.5.6'
-  s.add_dependency 'katello', '>= 4.8.0'
+  s.add_dependency 'foreman_remote_execution', '>= 9.0.0'
+  s.add_dependency 'katello', '>= 4.15.0'
```

Comment:
You can add a `gemfile.d/test.rb` like https://github.com/theforeman/foreman_rh_cloud/blob/develop/gemfile.d/katello.rb to test against Katello git instead of the released gem (A release with Rails 7 support won't happen in "the next days").

Reply:
Good hint, thanks.